### DISCUSSION TITLE
Debian security tracker - TEMP vulnerabilities

### DISCUSSION THREAD

**Discussion by SinghSarabpreet05** [aquasecurity/trivy#6455] [Q&A] [answered]
### Question

While scanning one of the debian images, some vulnerabilities are detected that are TEMP- and not CVEs. These are linked to debian security scanner.

For instance, running this scan outputs multiple TEMP- vulnerabilities:

`trivy image python:3.11.4-slim-bullseye`

One of those TEMPs are linked here: [TEMP-0841856-B18BAF](https://security-tracker.debian.org/tracker/TEMP-0841856-B18BAF)

Although these TEMPs can be ignored by setting `--severity MEDIUM,HIGH,CRITICAL` but that would ignore CVEs as well.

Is there a way to just ignore TEMP- vulnerabilities from showing up in the output?

### Target

Container Image

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Operating System

Linux

### Version

```bash
0.50.1
```

**Comment by knqyf263** [comment, marked as answer]
If you have a list of IDS, you can write them in .trivyignore.
https://aquasecurity.github.io/trivy/v0.50/docs/configuration/filtering/#by-finding-ids

If you want to ignore all CVEs prefixed TEMP-, you can write a Rego policy.
https://aquasecurity.github.io/trivy/v0.50/docs/configuration/filtering/#by-rego
